src/responsibleai/rai_analyse/rai_component_utilities.py

```diff
@@ -188,14 +203,15 @@ def load_dataset(dataset_path: str) -> pd.DataFrame:
             df = load_parquet(dataset_path)
             isLoadSuccessful = True
         except Exception as e:
-            new_e = UserConfigValidationException(
+            new_e = UserConfigError(
```

Comment:
We don't filter UserConfigError in dashboard -  dpv2 error table. It might show up again.

Reply:
thats okay, I can filter userconfig error in dashboard.